unit_tests/integration/conftest.py

```
@@ -252,7 +252,7 @@ def reload_config_for_test():
     docker_scylla.reload_config = reload_config_for_test
 
     cluster = docker_scylla.parent_cluster
-    os.environ.setdefault("_SCT_TEST_LOGDIR", "/tmp/test_vector_search_logs")
+    os.environ.setdefault("SCT_TEST_LOGDIR", "/tmp/test_vector_search_logs")
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_vector_search_logs"

(S108)